cmd/metamorph/main.go

```diff
@@ -55,11 +56,11 @@ func main() {
 	// setup signal catching
 	signalChan := make(chan os.Signal, 1)
 	signal.Notify(signalChan, os.Interrupt)
```

Comment:
maybe for conformity instead of `os.Interrupt` we could have `syscall.SIGINT` (they are the same)